src/containers/Network/BarChartVersion.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  Tooltip,
+  TooltipProps,
+  Label,
+  ResponsiveContainer,
+  Legend,
+  Cell,
+} from 'recharts'
+import {
+  ValueType,
+  NameType,
+} from 'recharts/src/component/DefaultTooltipContent'
+import './css/barchart.scss'
+
+interface Props {
+  data: any[]
+  stableVersion: string | null
+}
+
+interface BarCoordinates {
+  x: number
+  y: number
+}
+
+const CustomTooltip = ({
+  active,
+  payload,
+  label,
+}: TooltipProps<ValueType, NameType>) => {
+  const { t } = useTranslation()
+  if (active) {
+    return (
+      <div className="custom-tooltip">
+        <p className="label">{t('version_display', { version: label })}</p>
+        <p className="value">
+          {t('nodes_count', { count: payload?.[0].payload.count })}
```

Comment:
If there's no payload, should `count` be 0 instead of `undefined`? (Currently I think this will default to `undefined`)